common/app/views/fragments/headTonal.scala.html

```diff
@@ -85,10 +84,7 @@ <h1 class="content__headline js-score" itemprop="headline">
                     </div>
                 }
 
-                @if(
-                    (showBadge && !item.commercial.isAdvertisementFeature) ||
-                        (staticBadgesSwitch.isSwitchedOn && !item.tags.isVideo && !item.tags.isAudio)
-                ) {
+                @if(showBadge && !item.commercial.isAdvertisementFeature) {
```

Comment:
This is to avoid logos appearing twice in articles.  They appear in a different place on audio and video pages.  Without following the logic through completely I've just done what seems to make logos appear in the right places.